repos/Directory.Build.targets

```diff
@@ -86,6 +88,17 @@
     <WriteLinesToFile File="$(RepoCompletedSemaphorePath)ApplyPatches.complete" Overwrite="true" />
   </Target>
 
+  <Target Name="UpdateBuildToolFramework"
+    Condition="'$(EngCommonToolsShFile)' != ''"
+    Outputs="$(RepoCompletedSemaphorePath)UpdateBuildToolFramework.complete" >
+
+    <ReplaceTextInFile InputFile="$(EngCommonToolsShFile)"
+        OldText="_InitializeBuildToolFramework=&quot;netcoreapp2.1&quot;"
```

Comment:
Style nit: newline before first parameter for element in a target, for readability.